test/loading.jl

```diff
@@ -1466,7 +1467,6 @@ end
 
             log = @async String(read(out))
             @test success(proc)
-            fetch(log)
```

Comment:
Erroneous removal? Otherwise you don't return the log, but the test result, which is why CI fails:

```
  Test threw exception
  Expression: !(occursin(r"Loading (cache|object cache) file", log))
  MethodError: no method matching occursin(::Regex, ::Test.Pass)
```

Reply:
Yes, not sure what happened there..